src/Umbraco.Cms.Integrations.DAM.Aprimo/App_Plugins/UmbracoCms.Integrations/DAM/Aprimo/Render/AprimoAsset.cshtml

```diff
@@ -0,0 +1,9 @@
+﻿@inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage<Umbraco.Cms.Integrations.DAM.Aprimo.Models.ViewModels.AprimoAssetViewModel>
+
+<div>
```

Comment:
If the implementer was going to use this directly, they could benefit from some hooks for CSS - i.e. class name on the surrounding div, and removal of the inline styles.

Reply:
Removed

Reply:
Updated